apps/src/lab2/lab2Redux.ts

```diff
@@ -524,6 +528,31 @@ async function cleanUpProjectManager() {
   Lab2Registry.getInstance().clearProjectManager();
 }
 
+// Returns if the current state represents a predict level that should be read only.
+// If the predict level code is not editable after submit or the user has not submitted a response,
+// the predict level is read only.
+function isReadonlyPredictLevel(state: RootState) {
+  const isPredictLevel =
+    state.lab.levelProperties?.predictSettings?.isPredictLevel || false;
+  let isReadonlyPredictLevel = isPredictLevel;
```

Comment:
can't this just be

```
let isPredictLevel = state.lab.levelProperties?.predictSettings?.isPredictLevel || false;
```

it doesn't need the separate variable here, does it?

Reply:
yeah I can consolidate this

Reply:
Actually I think I'm going to leave as-is despite it being overly verbose. I could see it causing confusion to have
```
let isReadonlyPredictLevel = ...
if (isReadonlyPredictLevel) {
...
}
```
Because at the point of the if statement, `isReadonlyPredictLevel` is not an accurate description of the boolean, the boolean only represents if the level is a predict level.